### PR TITLE
Fix typos in LowLevelInfo

### PR DESCRIPTION
**Pull request by quink-black**

### CODE REVIEW

File: docs/LowLevelInfo.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 ## Introduction
 
-This documnent will contain loose information for various topis referring to
+This document will contain loose information for various topics referring to
 the SRT source code describing some cross-source analysis that would be
-unobvious for a source code reviewer. It's not a complete documentation of
+non-obvious for a source code reviewer. It's not a complete documentation of
```
Comment (maxsharabayko) [inline docs/LowLevelInfo.md:7]: Not sure about this change: "unobvious" -> "non-obvious". @stevomatthews what do you think?

Comment (stevomatthews) [reply docs/LowLevelInfo.md:7]: "that would not be obvious for a source code reviewer" is the safe way to go!  ;-)

File: docs/LowLevelInfo.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 ## Introduction
 
-This documnent will contain loose information for various topis referring to
+This document will contain loose information for various topics referring to
 the SRT source code describing some cross-source analysis that would be
```
Comment (stevomatthews) [inline docs/LowLevelInfo.md:6]:
```suggestion
This document contains information on various topics related to
the SRT source code, including descriptions of some cross-source analysis that would be
```

**Review by stevomatthews — COMMENTED** [review]
Minor suggestions

File: docs/LowLevelInfo.md (hunk 1 above)
Comment (maxsharabayko) [reply docs/LowLevelInfo.md:7]: @quink-black Please also apply this suggestion.

Comment (quink-black) [reply docs/LowLevelInfo.md]: Done. It's amazing I can learning English from this project : P

**Pull request merged**